src/main/java/ca/team2706/frc/robot/Robot.java

```diff
@@ -138,6 +135,8 @@ public void autonomousInit() {
         // Iterate through each of the state-change listeners and call them.
         onStateChange(RobotState.AUTONOMOUS);
 
+        logFMSData();
```

Comment:
Can we now keep this always on without adversely impacting system performance?

Reply:
Oh yeah, forgot to mention to you. We found out just before Ryerson that the reason the robot was overrunning the loop time wasn't logging but it was the Camera Server (somehow). We actually had logging on throughout the entire Ryerson competition.

Reply:
Oops nevermind, I thought this was the periodic logging of event data. My question still stands - just not here :)

Reply:
Great news about logging. I think we're going to look into moving both camera feeds to the Pi for better performance.